qa/restart-upgrade/build.gradle

```diff
@@ -25,34 +26,44 @@ testClusters {
 }
 
 // Task to run BWC tests against the old cluster
-task testAgainstOldCluster(type: StandaloneRestIntegTestTask) {
-    dependsOn "zipBwcPlugin"
-    useCluster testClusters."${baseName}"
-    systemProperty 'tests.rest.bwcsuite_cluster', 'old_cluster'
-    systemProperty 'tests.is_old_cluster', 'true'
-    systemProperty 'tests.skip_delete_model_index', 'true'
-    systemProperty 'tests.plugin_bwc_version', knn_bwc_version
-    nonInputProperties.systemProperty('tests.rest.cluster', "${-> testClusters."${baseName}".allHttpSocketURI.join(",")}")
-    nonInputProperties.systemProperty('tests.clustername', "${-> testClusters."${baseName}".getName()}")
-    systemProperty 'tests.security.manager', 'false'
-}
+    task testAgainstOldCluster(type: StandaloneRestIntegTestTask) {
+        dependsOn "zipBwcPlugin"
+        useCluster testClusters."${baseName}"
+        systemProperty 'tests.rest.bwcsuite_cluster', 'old_cluster'
+        systemProperty 'tests.is_old_cluster', 'true'
+        systemProperty 'tests.skip_delete_model_index', 'true'
+        systemProperty 'tests.plugin_bwc_version', knn_bwc_version
+        if (knn_bwc_version.startsWith("1.0") || knn_bwc_version.startsWith("1.1")) {
```

Comment:
Nice, this makes code much better. For filtering out tests, add a comment explaining why they are filtered out.